docs/guides/jest.md

```diff
@@ -2,7 +2,9 @@
 
 ## Configure with Jest
 
-To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` to literally the string `<rootDir>` and the path to your setup file.
+To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` in the `package.json` file of the project to literally the string `<rootDir>` and the path to your setup file.
```

Comment:
This isn't necessarily where the config lives - it might be in a separate config file.

Reply:
Would it be acceptable to change it to:

```To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` in your config file (e.g.: the `package.json` file of the project) to literally the string `<rootDir>` and the path to your setup file.```

My interpretation is that making this explicit can help people.

Reply:
that sounds good, but let’s also include an example that’s not package.json, so people will know that they have to find their config file themselves?

Reply:
How about:

>To run the setup file to configure Enzyme and the Adapter with Jest direct `setupTestFrameworkScriptFile` in your config file (check Jest's documentation for the [possibles locations of that config file](http://jestjs.io/docs/en/configuration)) to literally the string `<rootDir>` and the path to your setup file.

Reply:
That sounds great!